### ISSUE TITLE
Flow error observed very often

### ISSUE THREAD

**Issue by yzainee** [selinon/selinon#152]
## Issue description:
During the ingestion of epvs, there seems to be an error in celery which can be traced back to selinon dispatcher. Currently do not have much idea about the error or the root cause of the same. Need help around it.

### Links to logs:

https://errortracking.prod-preview.openshift.io/openshift_io/fabric8-analytics-production/issues/10377/?referrer=slack

**Comment by fridex** [comment]
Unfortunately, I cannot reach to the log. Could you please add me?

**Comment by fridex** [comment]
> Unfortunately, I cannot reach to the log. Could you please add me?

Or post the exception publicly.

**Comment by yzainee** [comment]
are you not able to login via github?

**Comment by yzainee** [comment]
![Screenshot from 2020-02-06 18-07-18](https://user-images.githubusercontent.com/37098367/73937757-c9f58480-490b-11ea-996a-7c6898b34910.png)

**Comment by fridex** [comment]
> are you not able to login via github?

I'm getting "Authentication error: You do not have access to the required GitHub organization.".

Is this specific to Celery or to Selinon? Seems like the exception is happening on celery layer.

**Comment by yzainee** [comment]
yup. From the exception, thats what it looks like. But thought probably you could have a look at it once to see if all is fine from selinon dispatcher side.

**Comment by yzainee** [comment]
which github repo are you trying to view?

**Comment by fridex** [comment]
> which github repo are you trying to view?

I'm trying to log into sentry.

![Screenshot from 2020-02-06 14-32-34](https://user-images.githubusercontent.com/880687/73941432-91df4900-48ed-11ea-98ed-588515106078.png)

> yup. From the exception, thats what it looks like. But thought probably you could have a look at it once to see if all is fine from selinon dispatcher side.

I'm not sure if this exception is related to sentry. What version of celery are you using? Can you try to downgrade to older version (e.g. 4.0)?

**Comment by yzainee** [comment]
This is what i see in the worker logs

++ selinon-cli version
--
  | Selinon version: 1.0.0
  | Celery version: 4.2.1

**Comment by fridex** [comment]
> This is what i see in the worker logs
> 
> ## ++ selinon-cli version
>   | Selinon version: 1.0.0
>   | Celery version: 4.2.1

Can you grant me access to sentry log? I think there is an option to export exception for non-sentry users.

**Comment by yzainee** [comment]
https://errortracking.prod-preview.openshift.io/share/issue/aec192d877ee42268205efe3794f0681/ can you try this link

**Comment by yzainee** [comment]
similar error: https://errortracking.prod-preview.openshift.io/share/issue/aec192d877ee42268205efe3794f0681/

**Comment by fridex** [comment]
This looks like an issue on Celery side when deserializing exception. It will be good to take a look if the given exception was raised when task fails or succeeds.

If the task is successful, it might be issue on task status handling for successful tasks.

If the task failed, it might be an issue on task exception retrieval from backend.

It's hard to debug for me just based on the traceback provided. Do you have a minimal reproducer for this?

**Comment by yzainee** [comment]
Well it would be difficult to provide a minimal reproducer. Would you be able to setup a dev cluster for dependency analytics? I can point you to the steps.. or else, if you can, you can use Michal's (Michal Srb) setup if its available.
